### DISCUSSION TITLE
Bulk request for linking records [May 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5741] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@253601577271689216> One option is to find associated tableId for junction table & to bulk write into it directly.    
####    
#### For Actor <many to many> Film, below is the junction table. You can enable junction table visibility from Project settings menu `Show M2m tables`  
![image0](https://cdn.discordapp.com/attachments/1096111769182810252/1096646455034392576/Screenshot_2023-04-15_at_9.29.15_AM.png)  
![image1](https://cdn.discordapp.com/attachments/1096111769182810252/1096646455265083442/Screenshot_2023-04-15_at_9.29.21_AM.png)  
![image2](https://cdn.discordapp.com/attachments/1096111769182810252/1096646455567065138/Screenshot_2023-04-15_at_9.29.29_AM.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### It's a work around. We will consider providing an bulk API for linking records when ever we take up updates to API next  
#  
> 🙋‍♂️ **spaceb0t**  
#### I put a note in <#661905455894888493> about how I was able to successfully batch update linked records via api using n8n  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1096111769182810252)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
